docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,130 @@
+
+Additionally, the `Digital signature and/or key agreement` must also be set with the extension value in the Key Usage section. For more information, see [key usage extensions and extended key usage](https://help.hcltechsw.com/domino/10.0.1/admin/conf_keyusageextensionsandextendedkeyusage_r.html).
+
+### Configuration in zowe.yaml file
+
+Zowe records the final configuration in `zowe.yaml` file.
+
+## How to manage certificates and verification in Zowe?
+
+Certificates and their verification are managed within a keystore (certificate storage) and a truststore (verification storage).
+
+Zowe supports certificates that are stored either in a USS directory Java KeyStore format or else held in a z/OS Keyring. z/OS keystore is the preferred choice for storing certificates where system programmers are already familiar with their operation and usage. The user ID setting up a keystore and connecting it with certificates requires elevated permissions, and in scenarios where you need to create a Zowe sandbox environment or for testing purposes and your TSO user ID doesn't have authority to manipulate key rings, USS keystores are a good alternative.
+
+* If you are using a USS keystore, then the script zowe-setup-certificates.env is the configuration step required to create the USS directory that contains the certificate. This is described in detail in Configuring Zowe certificates in a USS KeyStore.
+
+* If you are using a key ring, the sample JCL member `ZWEKRING` provided in the PDS library `SZWESAMP` contains the security commands to create a key ring and manage its associated certificates. This is described in Configuring Zowe certificates in a key ring which provides instructions for how to configure Zowe to work with the following certificates.
+    * a self-signed certificate
+    * a certificate signed with an existing certificate authority
+    * an existing certificate already held in the SAF database that can be added to the Zowe key ring.
+
+For both scenarios, where the certificate is held in a USS Java Keystore or a z/OS key ring, the `USS KEYSTORE_DIRECTORY` is still required which is created with the script zowe-setup-certificates.sh.
+
+* In the USS scenario, this directory holds the `.cer` and `.pem` files for the certificate itself.
+* In the key ring scenario, this directory stores the location and name of the Zowe key ring and its certificates.
+
+### Enable certificate validation without hostname validation
```

Comment:
To follow our style guide: Enabling certificate validation without hostname validation